lib/go-tc/parameters.go

```diff
@@ -210,10 +210,10 @@ func (pp *PostProfileParam) Validate(tx *sql.Tx) error {
 	}
 	if pp.ParamIDs == nil {
 		errs = append(errs, errors.New("paramIds missing"))
-	} else if ok, err := ParamsExist(*pp.ParamIDs, tx); err != nil {
+	} else if ok, nonExistingIDs, err := ParamsExist(*pp.ParamIDs, tx); err != nil {
 		errs = append(errs, errors.New(fmt.Sprintf("checking parameter IDs %v existence: "+err.Error(), *pp.ParamIDs)))
 	} else if !ok {
-		errs = append(errs, errors.New(fmt.Sprintf("parameters with IDs %v don't all exist", *pp.ParamIDs)))
+		errs = append(errs, errors.New(fmt.Sprintf("parameters with IDs %v don't all exist", nonExistingIDs)))
```

Comment:
Please use `fmt.Errorf(args)` instead of `errors.New(fmt.Sprintf(args))`. I see it being done in other places but if you've gotta change the line anyway, this heinous practice can be fixed in at least this one place.

also, the text "... don't all exist" implies that one or more of them may actually exist, but it's more accurate to report "... all don't exist" or just "... don't exist".